.github/workflows/main.yaml

```diff
@@ -25,6 +25,7 @@ env:
   # /make/builder.Dockerfile
   # /taprpc/Dockerfile
   # /tools/Dockerfile
+  # /.github/workflows/*.yml
```

Comment:
is `*.yml` going to match the `.yaml` files?

Reply:
It shouldn't, but this is only a reminder for future contributors.

Could probably script those golang version bumps tbh,